packages/widget-message-meet/src/components/activity-list/__snapshots__/index.test.js.snap

```diff
@@ -1,197 +1,8 @@
-          <span
-            className="avatar-letter">
-            T
-          </span>
-        </div>
-      </div>
-      <div
-        className="content-container">
-        <div
-          className="meta">
-          <div
-            className="display-name"
-            title="{name}">
-            Test User 2
-          </div>
-          <div
-            className="published">
-            9/20/16, 3:53 PM
-          </div>
-        </div>
-        <div
-          className="activity-content">
-          <div
-            className="activity-text"
-            dangerouslySetInnerHTML={
-              Object {
-                "__html": "Test Activity Content 2",
-              }
-            } />
-        </div>
-      </div>
-      <div
-        className="activity-post-actions">
-        <div
-          className="activity-post-action">
-          <div
-            className="post-action-item">
-            <button
-              className="button"
-              onClick={[Function]}
-              onKeyUp={[Function]}>
-              <div
-                className="icon"
-                title="Flag this message" />
-            </button>
-          </div>
-        </div>
-        <div
-          className="activity-post-action">
-          <div
-            className="action-spacer" />
-        </div>
-      </div>
-    </div>
-  </div>
-  <div
-    className="activity-item-container">
-    <div
-      className="activity-item">
-      <div
-        className="avatar-wrapper">
-        <div
-          className="avatar"
-          style={undefined}>
-          <span
-            className="avatar-letter">
-            T
-          </span>
-        </div>
-      </div>
-      <div
-        className="content-container">
-        <div
-          className="meta">
-          <div
-            className="display-name"
-            title="{name}">
-            Test User 3
-          </div>
-          <div
-            className="published">
-            9/20/16, 3:54 PM
-          </div>
-        </div>
-        <div
-          className="activity-content">
-          <div
-            className="activity-text"
-            dangerouslySetInnerHTML={
-              Object {
-                "__html": "Test Activity Content 3",
-              }
-            } />
-        </div>
-      </div>
-      <div
-        className="activity-post-actions">
-        <div
-          className="activity-post-action">
-          <div
-            className="post-action-item">
-            <button
-              className="button"
-              onClick={[Function]}
-              onKeyUp={[Function]}>
-              <div
-                className="icon"
-                title="Flag this message" />
-            </button>
-          </div>
-        </div>
-        <div
-          className="activity-post-action">
-          <div
-            className="action-spacer" />
-        </div>
-      </div>
-    </div>
-  </div>
+  
```

Comment:
Did we lose the snapshot again or is this what we talked about before with the list rendering?